test/distributed/test_distributed_spawn.py

```diff
@@ -25,9 +26,25 @@
     print("Spawn not available, skipping tests.", file=sys.stderr)
     sys.exit(0)
 
+_allowed_backends = ("gloo", "nccl", "ucc")
+if (
+    "BACKEND" not in os.environ
+    or "WORLD_SIZE" not in os.environ
+    or "TEMP_DIR" not in os.environ
+    or not path.exists(path.join(os.environ["TEMP_DIR"], "barrier"))
+):
+    # TODO can we actually have `run_tests.py` emit the complete instructions when it prints a repro command?
+    raise RuntimeError(
+        "Missing expected env vars for `test_distributed_spawn.py`.  Please ensure to specify the following:\n"
+        f"'BACKEND' = one of {_allowed_backends}\n"
+        f"'WORLD_SIZE' = int >= 2\n"
+        "'TEMP_DIR' specifying a directory containing a barrier file named 'barrier'.\n\n"
```

Comment:
maybe make the error message clear that `barrier` is inter-test barrier not the dist.barrier?

Reply:
oops i didn't notice this, well, i'm gonna leave it alone but if you want to change it ill stamp ;)